### PR TITLE
Template Updates: DIY 

### LINKED ISSUE #6971: Template Updates: DIY
## Description 

Edits in **BOLD** are new from April 6, 2021.

| Page | Animations | Cosmetics |
| --- | --- | --- |
| 1 |  headline updated to fly in; remove background image animation. **Issue No text to fly in**  | combine 'x' shape; change font colors.  |
| 2 | headline updated to fly in; remove background animation.  | change font colors. |
| 3 |  avatar updated to scale in instead of move/zoom. **Quote to fly in**  |  N/A  |
| 4 |  headline updated to fly in; remove background animation.  | change font colors.  |
| 5 | all text updated to fly in. **Image/Shape Fade in (3000ms)** | combine arrow shape; change color of fonts and rectangle shapes; change background color.  |
| 6 |  Swap plus sign and circle to pulse; headline changed to fly in; remove animations on background image and left band.   | combine + shape and change color. |
| 7 | text updated to fly in.  **Image/Shape Fade in (3000ms)** | Change colors of fonts and rectangle shapes; change background color.  |
| 8 | Headline, text, and images should change to fly in. **Tan shape Fade in (3000ms)** | Combine arrow shapes; change background color; change font color; change rectangle shape color. |
| 9 |  Headline change to fly in. **Image/Shape Fade in (3000ms)** | Combine dot shape; change font color; increate tappable spots. |
| 10 | Headline, Text, image change to fly in. **Background Shape Fade in (3000ms)** | combine social media shapes and change color; change background color; change font and rectangle shape color.  |

See Figma for specifications: https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories-Post-Stable?node-id=7216%3A549493 

If design requests have updated please make note in a comment here so that testing is aware of updated specs. 

## Acceptance 
- Template is updated 
- Relevant stickers are created 
- No more custom animations

### PR DESCRIPTION
**Pull request by samwhale**
## Context

Updating the templates to match [new designs](https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories-Post-Stable?node-id=7357%3A545144).

~Note: new animations are being added later, so those have not been implemented here.~ 
April 10, 2021: Custom animations have been removed for existing animations 

## Summary

Updates the DIY template.

## Relevant Technical Choices

n/a

## To-do

- [x] Design review

## User-facing changes

**Before:**

https://user-images.githubusercontent.com/22185279/113633370-1466b400-962a-11eb-97fd-0fe825f8295b.mov

**After:**

https://user-images.githubusercontent.com/22185279/113633367-12045a00-962a-11eb-8521-e6d3224c166f.mov

Should match these specifications:
image

## Testing Instructions

1. Select the template the `Explore Templates` section in the dashboard
2. Go to the preview
3. Watch the new template with animations
4. Make sure the colors and animations line up with the figma document - there should be no custom animations 

### QA

- [ ] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:
1. Select the template the `Explore Templates` section in the dashboard
2. Go to the preview
3. Watch the new template with animations

### UAT

- [x] UAT should use the same steps as above.

### Does this PR have a security-related impact?

no

### Does this PR change what data or activity we track or use?

no

### Does this PR have a legal-related impact?

no

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testiing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [ ] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #6971

### CODE REVIEW

**Comment by swissspidy** [comment]
Stickers are still behind a feature flag.

If we update the templates to use the new stickers feature, we should probably enable that flag by default ASAP.

FYI @BrittanyIRL

**Comment by BrittanyIRL** [comment]
> Stickers are still behind a feature flag.
> 
> If we update the templates to use the new stickers feature, we should probably enable that flag by default ASAP.
> 
> FYI @BrittanyIRL

We're just subbing out the existing "shapes" here for "stickers", like we did with the hearts on the beauty quotes. There's nothing really to interact with by anyone except pea pod implementing these. It's a bit of a test run with the whole thing. Unless I'm misreading what you are concerned about?

**Comment by swissspidy** [comment]
Ugh, apologies 🤦 I was worried that stickers as a whole are behind a feature flag. Turns out only the stickers _panel_ is. False alarm!

**Comment by BrittanyIRL** [comment]
> Ugh, apologies 🤦 I was worried that stickers as a whole are behind a feature flag. Turns out only the stickers _panel_ is. False alarm!

ohhh yeah that would be an issue! I'm glad you double checked and that it's a false alarm!

**Review by BrittanyIRL — APPROVED** [review]
LGTM! Tested when I pulled down to update the custom animations.

File: assets/src/edit-story/stickers/index.js [hunk 1]
```diff
@@ -18,7 +18,10 @@
  */
 import { default as sample } from './sampleSticker';
 import { default as beautyHeart } from './beautyHeart';
+import * as diyStickers from './diy';
+
 export default {
   sample,
   beautyHeart,
+  ...diyStickers,
```
Comment (maxyinger) [inline assets/src/edit-story/stickers/index.js]: yea I like how much this cleans it up. especially once we got a bunch of new stickers in here for the templates should make all the stickers much more organized 🎉

**Review by maxyinger — APPROVED** [review]
Looks great! I like the stickers organization

**Comment by swissspidy** [comment]
@BrittanyIRL could you please reply to the CLA bot since you made some changes here? thanks 🙂

**Comment by BrittanyIRL** [comment]
@googlebot I consent.

**Review by embarks — APPROVED** [review]

**Comment by ivayloklissarov** [comment]
Small fixes:

Page 3
- Remove animation from image and "anonymous" text
Screen Shot 2021-04-15 at 9 10 51 AM

Page 5
- Remove animation from CTA text
Screen Shot 2021-04-15 at 9 11 21 AM

Page 8
- Remove animation from CTA text
Screen Shot 2021-04-15 at 9 12 03 AM

**Comment by BrittanyIRL** [comment]
> Small fixes:
> 
> Page 3
> 
> * Remove animation from image and "anonymous" text

diy page 3


> Page 5
> 
> * Remove animation from CTA text

diy page 5


> Page 8
> 
> * Remove animation from CTA text

diy page 8

**Pull request merged**